labelbox/data/annotation_types/data/tiled_image.py

```diff
@@ -0,0 +1,364 @@
+import math
+import logging
+from enum import Enum
+from typing import Optional, List, Tuple
+from concurrent.futures import ThreadPoolExecutor
+
+import requests
+import numpy as np
+import tensorflow as tf
+from retry import retry
+from pyproj import Transformer
+from pydantic import BaseModel, validator, conlist
+from pydantic.class_validators import root_validator
+
+from ..geometry import Point
+from .base_data import BaseData
+from .raster import RasterData
+"""TODO: consider how to swap lat,lng to lng,lt when version = 2...
+should the bounds validator be inside the TiledImageData class then, 
+since we need to check on Version?
+"""
+
+VALID_LAT_RANGE = range(-90, 90)
+VALID_LNG_RANGE = range(-180, 180)
+TMS_TILE_SIZE = 256
+MAX_TILES = 300  #TODO: thinking of how to choose an appropriate max tiles number. 18 seems too small, but over 1000 likely seems too large
```

Comment:
@msokoloff1 is it okay to arbitrarily set this for now and see if we need to adjust it later on?